docs/src/content/hardhat-runner/docs/advanced/scripts.md

```diff
@@ -82,3 +74,5 @@ You can still pass arguments to Hardhat when using it as a library. This is done
 - `HARDHAT_VERBOSE`: Enables Hardhat verbose logging.
 
 - `HARDHAT_MAX_MEMORY`: Sets the maximum amount of memory that Hardhat can use.
+
+For example, instead of doing `npx hardhat --network localhost run script.js`, you should do `HARDHAT_NETWORK=localhost node script.js`.
```

Comment:
```suggestion
For example, instead of doing `npx hardhat --network localhost run script.js`, you can do `HARDHAT_NETWORK=localhost node script.js`.
```